pkg/utils/config.go

```diff
@@ -0,0 +1,133 @@
+package utils
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	color "github.com/threeal/bro/pkg/utils/color"
+)
+
+type Config interface {
+	Read() error
+	Write() error
+	init()
+}
+
+var HOME_DIR = getHomeDirectory()
+var BRO_CONFIG_DIR = filepath.Join(*HOME_DIR, ".bro")
```

Comment:
since `getHomeDirectory()` may return error, maybe we should make `BRO_CONFIG_DIR` as a function too.